ui/src/components/Arrow.tsx

```diff
@@ -0,0 +1,41 @@
+export function arrow(angle: number): string {
+  const rotation = angle + 140;
+  const unit = 100;
+
+  const rad = (angle * Math.PI) / 180;
+
+  const x = unit * Math.sin(rad);
+  const y = unit * Math.cos(rad);
+
+  const icon =
+    `
+    <svg width='70' height='70' viewBox='-10 -10 80 80' fill='none' xmlns='http://www.w3.org/2000/svg' >
+        <path transform='scale(0.3) translate(` +
+    y +
+    `, ` +
+    x +
```

Comment:
Can we do string interpolation with template literals here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

so:

```
<svg width='70' height='70' viewBox='-10 -10 80 80' fill='none' xmlns='http://www.w3.org/2000/svg' >
        <path transform='scale(0.3) translate(`${y}, ${x}) rotate(${rotation} blah blah
```